tests/test_pivot.py

```diff
@@ -136,16 +137,220 @@ def test_pivot_longer_rep_index_groups(self):
             stang,
             index_to = "idx",
             columns=["E_00","mu_00","E_45","mu_45","E_90","mu_90"],
```

Comment:
You could use the `gr.matches()` helper in place of enumerating the columns. This would simplify the test code, and check the functionality with the selection helpers.